### PR TITLE
Go Go Gadget Pull Request

### PR DESCRIPTION
**Pull request by UselessTheremin**
Adds a new "Inspector Gadget" fugitive (WIP)

:cl:  
rscadd: Inspector Gadget fugitive
/:cl:

### CODE REVIEW

**Comment by ChesterTheCheesy** [comment]
give him toolset arm implant please

**Comment by UselessTheremin** [comment]
> give him toolset arm implant please

added that and then saw your comment lol

**Comment by Tipy1802** [comment]
I know that on Goon the det's hat acts like Inspector Gadget's hat, may want to take a look at that

**Comment by athena413** [comment]
> 
> 
> I know that on Goon the det's hat acts like Inspector Gadget's hat, may want to take a look at that

goon code is copyrighted

**Comment by athena413** [comment]
template

File: code/modules/antagonists/fugitive/fugitive_outfits.dm [hunk 1]
```diff
@@ -112,3 +112,33 @@
...
 	W.update_label()
+
+/datum/outfit/gadget
+	name = "Inspector Gadget"
+	uniform = /obj/item/clothing/under/color/grey
+	suit = /obj/item/clothing/suit/det_suit/grey
+	head = /obj/item/clothing/head/fedora
+	shoes = /obj/item/clothing/shoes/sneakers/brown
+	ears = /obj/item/radio/headset
+	implants = list(/obj/item/implant/explosive, /obj/item/organ/cyberimp/arm/toolset)
```
Comment (alexkar598) [inline code/modules/antagonists/fugitive/fugitive_outfits.dm:123]: Aw hell naw, this is going to make the hunter's job miserable, if they kill him, his body disappers, if they dont kill him, he will blow up on you

**Pull request closed without merging**